src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
 
 <InstallFeedback />
 
+## Install Fluent Bit output plugin [#fluent-bit]
+
+New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
+
+You can install it in your cluster using our Helm chart.
```

Comment:
@paperclypse I'd use some h3s to clarify that you're offering two Helm approaches. To set that up, I'd suggest a sentence like this: "You can install it in your cluster using our Helm chart in two ways."